library/content_other/enchantment/src/main/java/org/quiltmc/qsl/enchantment/impl/EnchantmentGodClass.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.enchantment.impl;
+
+import org.jetbrains.annotations.ApiStatus;
+
+import org.quiltmc.qsl.enchantment.api.EnchantingContext;
+
+@ApiStatus.Internal
+public final class EnchantmentGodClass {
```

Comment:
Maybe something like `EnchantmentContextHolder`

Reply:
how dare you challenge the god class??? :p

Reply:
https://github.com/QuiltMC/quilt-standard-libraries/issues/13#issuecomment-881958085

It's for i5, and since it's impl, I think we can get away with it

Reply:
i mean, it has been almost 2 years since that PR was written; we should aim for the standards of today;
on a glance on that PR though, what is the point of the class though? can't it be merged with QuiltEnchantmentHelper?

Reply:
It can yeah, I only kept it to hide the threadlocal field and to keep `EnchantmentGodClass`